worlds/apsudoku/docs/setup_en.md

```diff
@@ -13,25 +11,33 @@ Does not need to be added at the start of a seed, as it does not create any slot
 
 ## Installation Procedures
 
-Go to the latest release from the [APSudoku Releases page](https://github.com/EmilyV99/APSudoku/releases). Download and extract the `APSudoku.zip` file.
+Go to the latest release from the [APSudoku Releases page](https://github.com/APSudoku/APSudoku/releases/latest). Download and extract the appropriate file for your platform.
 
 ## Joining a MultiWorld Game
 
-1. Run APSudoku.exe
-2. Under the 'Archipelago' tab at the top-right:
-	- Enter the server url & port number
+1. Run the APSudoku .exe (name varies by platform)
+2. Under `Settings` &rarr; `Connection` at the top-right:
+	- Enter the server IP & port number
 	- Enter the name of the slot you wish to connect to
 	- Enter the room password (optional)
 	- Select DeathLink related settings (optional)
-	- Press connect
-3. Go back to the 'Sudoku' tab
-	- Click the various '?' buttons for information on how to play / control
-4. Choose puzzle difficulty
-5. Try to solve the Sudoku. Click 'Check' when done.
+	- Press `Connect`
+3. Optional things:
+	- You can check other settings under `Settings` &rarr; `Sudoku`, and can change the colors used under `Settings` &rarr; `Theme`.
+	- While connected, you can view the `Console` and `Hints` tabs for standard TextClient like features
+	- You can also use the `Tracking` tab to view either a basic tracker, or a valid [GodotAP tracker pack](https://github.com/EmilyV99/GodotAP/blob/main/tracker_packs/GET_PACKS.md)
+4. Under the `Sudoku` tab
+	- While connected, the number of 'unhinted' locations for your slot is shown in the upper-left. (If this reads 0, no further hints can be earned for this slot, as every locations is already hinted)
+	- Click the various `?` buttons for information on how to play / control
+	- Choose puzzle difficulty
+	- Click `Start` to generate a puzzle
+5. Try to solve the Sudoku. Click `Check` when done.
+	- Correct solution rewards you with 1 hint for a location in the world you are connected to.
+	- Incorrect solution has no penalty, unless DeathLink is enabled (see below)
```

Comment:
All the rest of these use full sentences
```suggestion
	- A correct solution rewards you with 1 hint for a location in the world you are connected to.
	- An incorrect solution has no penalty, unless DeathLink is enabled (see below)
```

Reply:
conflicts with your other suggestion so I can't just accept it, bleh